src/MemoryForm/MemoryForm.js

```diff
@@ -8,55 +8,73 @@ import {
   Modal,
   Button,
   TouchableOpacity,
-  TextInput,
+  TextInput
 } from "react-native";
 import { editMemory } from "../apiCalls";
+// import { TextInput } from 'react-native-paper';
+import { Colors } from "react-native/Libraries/NewAppScreen";
 
 
 // Edit is clicked from within a memory
 // Flip a bool in Memory when edit mode is activated and
 // form will render instead of static memory
 
 
-export default MemoryForm = ({memory}) => {
+export default MemoryForm = ({memory, setSelectedMemory, setModalVisible, modalVisible, setEditMode, editMode}) => {
   const [updatedDescriptionText, setUpdatedDescriptionText] = useState(memory.description);
   const [updatedAromaText, setUpdatedAromaText] = useState(memory.aromas);
   const [updatedMemory, setUpdatedMemory] = useState({});
-  const [modalVisible, setModalVisible] = useState(true);
-
+  // const [formVisible, setFormVisible] = useState(true);
+  
   useEffect(() => {
+    // console.log('updatedMemory :>> ', updatedMemory);
+    console.log('updatedDescriptionText :>> ', updatedDescriptionText);
+    console.log('updatedAromaText :>> ', updatedAromaText);
     if (Object.keys(updatedMemory).length) {
+      console.log('if there is a new memory', updatedMemory);
       editMemory(memory.id, updatedMemory)
       .then(response => console.log(response))
+      setSelectedMemory(null);
+      setModalVisible(!modalVisible);
```

Comment:
This is a great start, however, we should consider giving a notification to the individual that the memory has been updated.  I also don't believe pushing a user back to the rooms page is the best way to handle the updated memory.  For now, this is good but keep this in the back of your mind!